README.md

```
@@ -1,70 +1,61 @@
-
-<br>
+| 운세 제공 | 부적 제공                                                                      |
+|--------|----------------------------------------------------------------------------|
+| ![운세 제공](docs/images/demo/fortune_overview.gif) | ![부적 제공](docs/images/demo/talisman_reward.gif) |
 
 ## 👨‍👦‍👦 팀원
 
-|<img src="https://avatars.githubusercontent.com/DongChyeon" width="150" height="150"/>|<img src="https://avatars.githubusercontent.com/Moonsukang" width="150" height="150"/>|
-|:-:|:-:|
-|Donghyeon Kim<br/>[@DongChyeon](https://github.com/DongChyeon)|Moonsu Kang<br/>[@MoonsuKang](https://github.com/MoonsuKang)|
+| <img src="https://avatars.githubusercontent.com/DongChyeon" width="150" height="150"/> | <img src="https://avatars.githubusercontent.com/Moonsukang" width="150" height="150"/> |
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)